pkgs/by-name/la/lazylibrarian/package.nix

```
@@ -0,0 +1,134 @@
+{
+  calibre,
+  fetchFromGitLab,
+  fetchPypi,
+  ffmpeg-headless,
+  lib,
+  nix-update-script,
+  python3Packages,
+  calibreSupport ? true,
+  ffmpegSupport ? true,
+}:
+let
+  ez_setup = python3Packages.buildPythonPackage rec {
+    pname = "ez_setup";
+    version = "0.9";
+    pyproject = true;
+
+    src = fetchPypi {
+      inherit pname version;
+      hash = "sha256-MDxbF9VS0eP7BQXYBUn4V59VfhPY3JDl7O88B9f1hkI=";
+    };
+
+    build-system = with python3Packages; [
+      distutils
+      setuptools
+      wheel
+    ];
+
+    pythonImportsCheck = [
+      "ez_setup"
+    ];
+  };
+
+  slskd-api = python3Packages.buildPythonPackage rec {
+    pname = "slskd-api";
+    version = "0.1.5";
+    pyproject = true;
+
+    src = fetchPypi {
+      inherit pname version;
+      hash = "sha256-LmWP7bnK5IVid255qS2NGOmyKzGpUl3xsO5vi5uJI88=";
+    };
+
+    build-system = with python3Packages; [
+      setuptools
+      setuptools-git-versioning
+      wheel
+    ];
+
+    dependencies = with python3Packages; [
+      requests
+    ];
+
+    pythonImportsCheck = [
+      "slskd_api"
+    ];
+  };
+in
+python3Packages.buildPythonApplication rec {
+  pname = "lazylibrarian";
+  version = "0-unstable-2025-07-08";
```

[pyproject.toml](https://gitlab.com/LazyLibrarian/LazyLibrarian/-/blob/master/pyproject.toml?ref_type=heads#L10)

```suggestion
  version = "2025.01.09-unstable-2025-07-08";
```